lib/assembly/svg-object-fns/create-svg-objects-from-assembly-component.ts

```diff
@@ -16,55 +16,59 @@ export function createSvgObjectsFromAssemblyComponent(
   transform: Matrix,
   firstPin: Point,
   name?: string,
+  ftype?: string,
 ): SvgObject {
   const { center, width, height, rotation = 0, layer = "top" } = component
   const [x, y] = applyToPoint(transform, [center.x, center.y])
   const [pinX, pinY] = applyToPoint(transform, [firstPin.x, firstPin.y])
   const scaledWidth = width * Math.abs(transform.a)
   const scaledHeight = height * Math.abs(transform.d)
 
+  const isChip = ftype?.includes("chip")
```

Comment:
this isn't the right check, although this is definitely the right idea.

There is a new circuit json property called `are_pins_interchangeable`, that is what you should use. If something has interchangeable pins, it should not have a triangle.

See https://github.com/tscircuit/circuit-json/pull/180

tscircuit core does not currently generate this property